.travis.yml

```diff
@@ -20,6 +20,8 @@ matrix:
     - { python: "3.6", env: TOXENV=py36-django20 }
     - { python: "3.5", env: TOXENV=py35-django21 }
     - { python: "3.6", env: TOXENV=py36-django21 }
+    - { python: "3.5", env: TOXENV=py35-django22 }
+    - { python: "3.6", env: TOXENV=py36-django22 }
```

Comment:
shouldn't you also add python 3.7 to the test matrix?

Reply:
yes, right now fixing the existing build erros are first prio, sure will add after fixing em